graylog2-server/src/main/java/org/graylog2/plugin/database/users/User.java

```diff
@@ -75,6 +79,12 @@
 
     void setEmail(String email);
 
+    void setFullName(String firstName, String lastName);
```

Comment:
> Is this an expected behaviour? Since we can't migrate users in a useful way automatically, I would expect them to at least being able to log in, even if a warning is shown on screen or the log. Another thing to consider is if this will also affect system users while doing their tasks.

@edmundoa This is definitely not expected. Thanks for finding it. I will fix it.

User tasks throughout Graylog should continue to work, since a `full_name` will always be persisted along with the first/last name for all users (composed from the first/last name).